speech/api/speech_async_grpc.py

```diff
@@ -59,8 +59,12 @@ def main(input_uri, encoding, sample_rate):
     # https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1beta1/cloud_speech.proto
     response = service.AsyncRecognize(cloud_speech_pb2.AsyncRecognizeRequest(
         config=cloud_speech_pb2.RecognitionConfig(
-            encoding=encoding,
-            sample_rate=sample_rate,
+            # There are a bunch of config options you can specify. See
+            # https://g.co/cloud/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig  # noqa
+            # for the full list.
+            encoding=encoding,  # one of LINEAR16, FLAC, MULAW, AMR, AMR_WB
+            sample_rate=sample_rate,  # the rate in Hz
```

Comment:
"hertz" is fine.

On Fri, Jul 22, 2016, 5:37 PM Jerjou <notifications@github.com> wrote:

> In speech/api/speech_async_grpc.py
> <https://github.com/GoogleCloudPlatform/python-docs-samples/pull/429#discussion_r71962848>
> :
>
> > @@ -59,8 +59,12 @@ def main(input_uri, encoding, sample_rate):
> >      # https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1beta1/cloud_speech.proto
> >      response = service.AsyncRecognize(cloud_speech_pb2.AsyncRecognizeRequest(
> >          config=cloud_speech_pb2.RecognitionConfig(
> > -            encoding=encoding,
> > -            sample_rate=sample_rate,
> > +            # There are a bunch of config options you can specify. See
> > +            # https://g.co/cloud/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig  # noqa
> > +            # for the full list.
> > +            encoding=encoding,  # one of LINEAR16, FLAC, MULAW, AMR, AMR_WB
> > +            sample_rate=sample_rate,  # the rate in Hz
>
> Hrm... is it capitalized when spelled out? Makes it look like the car
> company...
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/python-docs-samples/pull/429/files/4462b6f217dbc70d289a167efc5453a4b22d0282#r71962848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPUc4Li4hF7CYQmYg-WSirUznQ6-DA3ks5qYWI9gaJpZM4JTLlG>
> .
>